### PR TITLE
v4.19.0.4

### PR DESCRIPTION
**Pull request by schuylermartin45**
Worth noting here that I made sure that the license information matched the other `typeshed` projects.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
     - if not exist %SP_DIR%\\jsonschema-stubs\\__init__.pyi exit 1  # [win]
-  requires:
-    - pip
+
 
 about:
   home: https://github.com/python/typeshed
-  summary: Typing stubs for jsonschema
-  license: Apache-2.0
+  license: Apache-2.0 AND MIT
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:44]: The license provided on PyPI is Apache-2.0, see https://inspector.pypi.io/project/types-jsonschema/4.19.0.4/packages/01/1e/7bb26b307a3cd32da256ac6c5d60ccd8d01bb5e3c44e6c08bce680a765c5/types_jsonschema-4.19.0.4-py3-none-any.whl/types_jsonschema-4.19.0.4.dist-info/METADATA#line.6.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,41 +1,57 @@
...
-  - url: https://raw.githubusercontent.com/python/typeshed/main/LICENSE
-    sha256: 295f8538c94ae5c3043301cf7cff1c852dab6a786a8ddee471e061b40d5ecabe
+
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<36]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:17]:
```suggestion
  skip: true  # [py<38]
```
See https://inspector.pypi.io/project/types-jsonschema/4.19.0.4/packages/92/f0/40eee69d627ced6547ca68942b50f333638250f414e6f76faffd2a2dff04/types-jsonschema-4.19.0.4.tar.gz/types-jsonschema-4.19.0.4/setup.py#line.49

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,41 +1,57 @@
...
+  license: Apache-2.0 AND MIT
   license_file: LICENSE
+  license_family: OTHER
+  summary: Typing stubs for jsonschema
+  description: |
+    This is a PEP 561 type stub package for the jsonschema package. It can be
+    used by type-checking tools like mypy, pyright, pytype, PyCharm, etc. to
+    check code that uses jsonschema.
+  dev_url: https://github.com/python/typeshed
+  doc_url: https://json-schema.org/learn/getting-started-step-by-step
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:53]: I would probably point to https://python-jsonschema.readthedocs.io.

File: recipe/meta.yaml (hunk 1 above)
Comment (schuylermartin45) [reply recipe/meta.yaml:44]: I think PyPi is wrong. This is part of the [typeshed](https://github.com/python/typeshed) repo and that contains `Apache-2.0 AND MIT`.

`types-request` and `types-pyyaml` are also from `typeshed` and use this license.

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:44]: I am considering the source of truth to be the archive we download. If they make an error on the license in the PyPI metadata that's not our problem.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**